DSN/src/main/java/org/gfa/dsn/controller/UserController.java

```diff
@@ -0,0 +1,68 @@
+package org.gfa.dsn.controller;
+
+import org.gfa.dsn.dto.*;
+import org.gfa.dsn.repository.UserRepository;
+import org.gfa.dsn.service.JwtService;
+import org.gfa.dsn.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1")
+public class UserController {
```

Comment:
This should be named UserRestController. In this case, it won't be misleading, that this is not Controller but a RestController.

Reply:
Renamed